src/pages/Polling.js

```diff
@@ -175,27 +182,30 @@ function Polling({
           <ReactMarkdown
             className="markdown"
             skipHtml={true}
-            source={proposal.about}
+            source={poll.content}
           />
         </DescriptionCard>
         <RightPanels>
-          <VotingPanel proposal />
+          <VotingPanel poll={poll} />
           <DetailsPanelCard style={{ padding: '0px 30px 15px 30px' }}>
             <CardTitle>Details</CardTitle>
             {[
               {
                 name: 'Source',
                 component: (
                   <ExternalLink
-                    href={ethScanLink(proposal.source, network)}
+                    href={ethScanLink(poll.source, network)}
```

Comment:
The mockups show a source field, but since we're not actually deploying any contracts we won't have an etherscan link. Should this be replaced with the `discussion_link` from the CMS, or maybe the `multihash`?

Reply:
yeah, I think something like `document hash` or `poll details hash` makes sense. We might also add a `download document` field so that people can download the full document (front-matter and all) if they want to verify the hash for themselves.